src/redux/basket/types.ts

```diff
@@ -0,0 +1,13 @@
+export type BasketItem = {
```

Comment:
it is the better approach to use Interface instead of type

Reply:
Done.